bridges/snowbridge/pallets/outbound-queue-v2/src/test.rs

```diff
@@ -140,67 +140,6 @@ fn process_message_fails_on_overweight_message() {
 	})
 }
 
-#[test]
-fn governance_message_not_processed_in_same_block_when_queue_congested_with_low_priority_messages()
```

Comment:
why is this test obsolete?

Reply:
@vgeddes @yrong is this test truly obsolete? Is the functionality of governance messages not the same as Snowbridge v1?